ddtrace/internal/telemetry/data.py

```diff
@@ -56,9 +56,12 @@ def _get_application(key):
     This helper packs and unpacks get_application arguments to support caching.
     Cached() annotation only supports functions with one argument
     """
+    # avoid circular dependency
+    from ddtrace.internal import process_tags
```

Comment:
This is not the best way of avoiding circular references. Let's try to find a different approach where the dependency goes in only one direction.